Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -278,6 +277,8 @@ namespace EMStudio
             AtomToolsFramework::ModularViewportCameraControllerRequestBus::Event(
                 GetViewportId(), &AtomToolsFramework::ModularViewportCameraControllerRequestBus::Events::SetCameraPivotAttached,
                 m_renderer->GetCharacterCenter());
+
+            m_renderer->UpdateGroundplane();
```

Comment:
In theory the character can still run away from the ground in this case.
Scenario: Camera not following character, character running away from the center and further from the ground. Do we want to see 
1) Ground moving around with the character 
2) Ground stay still and the character eventually run off the grid

I think this change is ok, user probably expected to run off the grid in this case.